atlas_commons/utils.py

```diff
@@ -206,21 +207,39 @@ def create_layered_volume(
 
     assert_metadata_content(metadata)
 
+    if "ignore" in metadata:
+        ignore_ids_ = region_map.find(
```

Comment:
For a better readability, define a
`metadata_ignore = metadata["ignore"]
`
as in [here](https://github.com/BlueBrain/atlas-commons/blob/af02b8f9e71ae5883218e732e1568a95182c6ec9/atlas_commons/utils.py#L227), and use it hereinafter instead of `metadata["ignore"]`

Reply:
ok I added `metadata_ignore` as you suggested